### PR TITLE
Add filemanager plugin and update hugo picture

### PR DESCRIPTION
**Pull request by hacdias**
That ^

### CODE REVIEW

**Comment by mholt** [comment]
Nice, thanks!! It would probably be important to mention in the top of the docs that the user should cover it with some kind of authentication, even `basicauth`.

**Comment by hacdias** [comment]
Oh sure! :D

File: public/docs/filemanager.html [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+				<div class="heading"><i class="fa fa-plus-circle"></i> Addon</div>
+				This directive is a Caddy extension. To get it, select this feature when you <a href="/download">download</a> Caddy. Questions should be directed to its maintainer. <a href="https://github.com/hacdias/caddy-filemanager">github.com/hacdias/caddy-filemanager</a>
+			</div>
+
+			<p>
+				filemanager is an extension based on browse middleware. It provides a file managing interface within the specified directory and it can be used to upload, delete, preview and rename your files within that directory.
+			</p>
+			
+			<p>
+				It is extremely important for security reasons to cover the path of filemanager with some kind of authentication. You can use, for exemple, <a href="../basicauth"><code>basicauth</code></a> directive.
```
Comment (mholt) [inline public/docs/filemanager.html]: "for **example**"

Comment (hacdias) [inline public/docs/filemanager.html]: I'm joining portuguese with english! Awesome! ahahah. Thanks  :D

**Comment by mholt** [comment]
Thanks Henrique! It'd be really cool if there was a screenshot. :smiley: Especially with your updated UI that you showed me in Slack the other day.

Do you want to add one? Be sure to run it through tinypng.com first. ;)

**Comment by hacdias** [comment]
Hello!!

Yup, I'll add a picture, but only after fishing this. I have to review the hugo docs too. :)

**Comment by hacdias** [comment]
Hey @mholt 

I've changed the hugo pic and added a new one to filemanager. Yes, I passed them through tinypng.com :D See them and tell me if you like it.

**Comment by mholt** [comment]
Beautiful, thanks!

**Comment by hacdias** [comment]
Eheh you're welcome!

**Comment by MSF-Jarvis** [comment]
The site hasn't been built yet, I can't download another fantastic plugin from @hacdias after having been a fan `hugo`, this has been a shame

**Comment by hacdias** [comment]
Hey @MSF-Jarvis! I'm glad you liked ```hugo``` plugin. Thanks! The new version of ```caddyserver.com``` website will only be released after announcing (officially) the final build of Caddy 0.9. Anyway, both ```filemanager``` and ```hugo``` are not totally finished yet.

**Comment by MSF-Jarvis** [comment]
I hate the waiting game :sob:

**Pull request merged**